src/lib/runtime_config/runtime_config.ml

```diff
@@ -1036,6 +1158,105 @@ let combine t1 t2 =
   ; epoch_data = opt_fallthrough ~default:t1.epoch_data t2.epoch_data
   }
 
+let gen =
+  let open Quickcheck.Generator.Let_syntax in
+  let%map daemon = Daemon.gen
+  and genesis = Genesis.gen
+  and proof = Proof_keys.gen
+  and ledger = Ledger.gen in
+  { daemon = Some daemon
+  ; genesis = Some genesis
+  ; proof = Some proof
+  ; ledger = Some ledger
+  ; epoch_data =
+      None (* Leave it empty until we know better what should go there. *)
+  }
+
+let ledger_accounts (ledger : Mina_ledger.Ledger.Any_ledger.witness) =
+  Mina_ledger.Ledger.Any_ledger.M.foldi ~init:[]
```

Comment:
You are just using the `foldi` here to return a list of accounts. Then why not use the `to_list_sequential` function?

Reply:
Indeed, see [27157c9](https://github.com/MinaProtocol/mina/pull/14285/commits/27157c9f2cfccea40304fcdd826197888936ee5d).